src/lang.rs

```diff
@@ -138,6 +146,13 @@ impl R {
             _ => todo!(),
         }
     }
+
+    pub fn get_vec_string(&self) -> Vec<String> {
+        match self {
+            R::Vector(v) => v.clone().into(),
+            _ => todo!(),
+        }
+    }
```

Comment:
Good catch Doug, thank you. Since now we have the into method we can use that in the test. Just destructure the `R::Vector` then use `.into` 💯. I will adjust the tests as well.